mp/src/game/client/neo/ui/neo_hud_ammo.cpp

```diff
@@ -137,169 +123,170 @@ void CNEOHud_Ammo::DrawAmmo() const
 		wepName[textLen] = toupper(wepName[textLen]);
 	}
 	g_pVGuiLocalize->ConvertANSIToUnicode(wepName, unicodeWepName, sizeof(unicodeWepName));
-	Color box_color = Color(box_color_r, box_color_g, box_color_b, box_color_a);
-	Color ammo_color = Color(ammo_color_r, ammo_color_g, ammo_color_b, ammo_color_a);
+
 	DrawNeoHudRoundedBox(xpos, ypos, xpos + wide, ypos + tall, box_color, top_left_corner, top_right_corner, bottom_left_corner, bottom_right_corner);
 
 	surface()->DrawSetTextFont(m_hSmallTextFont);
-	surface()->DrawSetColor(ammo_color);
 	surface()->DrawSetTextColor(ammo_color);
-
 	int fontWidth, fontHeight;
 	surface()->GetTextSize(m_hSmallTextFont, unicodeWepName, fontWidth, fontHeight);
 	surface()->DrawSetTextPos((text_xpos + xpos) - fontWidth, text_ypos + ypos);
 	surface()->DrawPrintText(unicodeWepName, textLen);
 
 	if(dynamic_cast<C_WeaponGhost*> (activeWep))
```

Comment:
`if (activeWep->IsGhost())` is faster, because we skip the extra runtime dynamic_cast